compiler/src/vtable.sk

```diff
@@ -660,21 +651,14 @@ private fun createGCType(
         }
       }
     | Some(slotInfo) ->
-      // Set the GC bits. All pointers are at the beginning of each slot,
-      // but we tile it out as many times as will fit in a 64-bit word
-      // so the inner loop of the GC can be tighter.
-      for (i in Range(0, tilesPerMask)) {
-        slotInfo.types.eachWithIndex((idx, x) -> {
-          scalarType = x.getScalarType(env);
-          if (scalarType.isGCPointer()) {
-            bitOffset = slotInfo.bitOffsets[idx];
-            setBit(
-              (slotInfo.bitSize * i) + bitOffset,
-              !isAllDeepFrozen && !x.isDeepFrozen(),
-            )
-          }
-        });
-      }
+      // Possible optimization: tile the mask so the inner loop of the GC is tighter.
+      slotInfo.types.eachWithIndex((idx, x) -> {
+        scalarType = x.getScalarType(env);
+        if (scalarType.isGCPointer()) {
+          bitOffset = slotInfo.bitOffsets[idx];
+          setBit(bitOffset, !isAllDeepFrozen && !x.isDeepFrozen())
+        }
+      })
```

Comment:
If _the inner loop of the GC_ refers to what is in `SKIP_intern_array` then the tiling is not used as of today. I could rewrite it so that it's used btw, I doubt tough that it makes things really faster.

Reply:
IIUC, there used to be, or was an attempt to, tile out the refMask as much as possible in a 64-bit mask to avoid rereading it from the beginning when processing arrays. But no runtime function processing arrays make use of it.

Reply:
Ok thanks for checking, it's totally plausible that this is a remnant of the old GC.